EIPS/eip-3475.md

```diff
@@ -2,42 +2,40 @@
 description: Interface for creating tokenized obligations with abstract on-chain metadata storage
-author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
+author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold Edoumou (@Edoumou), Toufic Batrice (@toufic0710)
 discussions-to: https://ethereum-magicians.org/t/eip-3475-multiple-callable-bonds-standard/8691
-status: Last Call
-last-call-deadline: 2022-07-30
+status: Final
 type: Standards Track
 category: ERC
 created: 2021-04-05
+requires: 20, 721, 1155
 ---
 
 ## Abstract
 
 This API standard allows the creation of tokenized obligations with abstract on-chain metadata storage. Issuing bonds with multiple redemption data cannot be achieved with existing token standards. This proposal enables each bond class ID to represent a new configurable token type and corresponding to each class, corresponding bond nonces to represent an issuing condition or any other form of data in uint256. Every single nonce of a bond class can have its metadata, supply, and other redemption conditions.
 
 ## Motivation
+Current LP (Liquidity Provider) tokens are simple [EIP-20](./eip-20.md) tokens with no complex data structure. To allow more complex reward and redemption logic to be stored on-chain, we need a new token standard that:
 
+ - Supports multiple token IDs
+ - Can store on-chain metadata
+ - Doesn't require a fixed storage pattern
+ - Is gas-efficient.
 
-Current LP tokens are simple [EIP-20](./eip-20.md) tokens with no complex data structure. To allow more complex reward and redemption logic to be stored on-chain, we need a new token standard that can manage:
+This EIP allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens' balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
 
- - Multidimensional token ID.
- - Store on-chain metadata.
- - Non-specified and flexible storage.
- - Gas efficient.
+The reason behind the standard's name (abstract storage bond) is its ability to store all the specifications (metadata/values and transaction as defined in the following sections) without needing external storage on-chain/off-chain. 
 
-This proposal allows the creation of any obligation with the same interface. It will enable any 3rd party wallet applications or exchanges to read these tokens' balance and redemption conditions. These bonds can also be batched as tradeable instruments. Those instruments can then be divided and exchanged in secondary markets.
+The functions defined in this EIP provide a gas-efficient solution for creating, trading, and burning bonds. They offer more optimization options for existing AMM pools.
```

Comment:
```suggestion
```
If anything, this should be moved to the rationale and mentioned as a trade-off.